cpp/ql/src/Metrics/Files/FNumberOfTests.ql

```diff
@@ -18,6 +18,9 @@ Expr getTest() {
   or
   // boost tests; http://www.boost.org/
   result.(FunctionCall).getTarget().hasQualifiedName("boost::unit_test", "make_test_case")
+  or
+  // googletest tests; https://github.com/google/googletest/
+  result.(FunctionCall).getTarget().hasQualifiedName("::testing::internal::", "MakeAndRegisterTestInfo")
```

Comment:
```suggestion
  result.(FunctionCall).getTarget().hasQualifiedName("testing::internal", "MakeAndRegisterTestInfo")
```

This string has to match exactly how we pretty-print the namespace, so it couldn't have worked with the leading and trailing colons. Can you point me to a project, ideally on lgtm.com, where we can test that the matching works as intended?

Reply:
With your correction I tested on leveldb.
See: https://lgtm.com/query/4423071705091911957/